velox/expression/CastExpr-inl.h

```diff
@@ -372,7 +386,7 @@ void CastExpr::applyCastKernel(
     }
 
   } catch (const VeloxException& ue) {
-    if (!ue.isUserError()) {
+    if (!ue.isUserError() || !wrapException) {
```

Comment:
In which case wrapException is false here? We only set it to false in setError? Thanks!

Reply:
It would be false if 'setError' wasn't called. This happens if an exception is raised and is handled in 'catch'.